apps/main/src/app/Home/Home.tsx

```diff
@@ -1,375 +1,121 @@
-              <li>Start the MariaDB and Redis services.</li>
-            </ul>
-          </div>
+        {/* Divider za Nastavitve */}
+        <p className="text-sm text-gray-500 px-3 mt-4">Nastavitve</p>
 
-          {/* Step 3 */}
-          <div>
-            <h3 className="text-xl font-semibold mb-2">3. Verify the Deployment</h3>
-            <p className="mb-2">
-              After starting the services, verify that everything is running correctly:
-            </p>
-            <ul className="list-disc list-inside mt-2">
-              <li>Access the frontend at <code>http://&lt;your-server-ip&gt;</code>.</li>
-              <li>Access the backend API at <code>http://&lt;your-server-ip&gt;:8000/api</code>.</li>
-            </ul>
-          </div>
+        {SETTINGS_ITEMS.map((item) => (
+          <SidebarButton key={item.name} item={item} activeTab={activeTab} setActiveTab={setActiveTab} />
+        ))}
+      </nav>
 
-          {/* Step 4 */}
-          <div>
-            <h3 className="text-xl font-semibold mb-2">4. Stop the Production Services</h3>
-            <p className="mb-2">
-              To stop the production services, run:
-            </p>
-            <pre className="bg-gray-100 p-4 rounded-lg">
-              <code>docker-compose -f docker-compose.prod.yml down</code>
-            </pre>
-            <p className="mt-2">
-              This will stop and remove all containers, networks, and volumes associated with the production deployment.
-            </p>
-          </div>
-        </div>
+      {/* Uporabniški meni */}
+      <UserSection />
+    </div>
+  );
+}
 
-        {/* Key Differences in docker-compose.prod.yml */}
-        <div className="mt-6">
-          <h3 className="text-xl font-semibold mb-2">Key Differences in <code>docker-compose.prod.yml</code></h3>
-          <p className="mb-2">
-            The <code>docker-compose.prod.yml</code> file is optimized for production with the following changes:
-          </p>
-          <ul className="list-disc list-inside space-y-2">
-            <li><strong>Frontend</strong>: Uses a production build of the React app, served via a lightweight web
-              server <strong>nginx</strong>.
-            </li>
-            <li><strong>Backend</strong>: Runs the Express app with <code>NODE_ENV=production</code> for better
-              performance and security.
-            </li>
-            <li><strong>Database</strong>: Persists MariaDB data to a named volume to ensure data is not lost when
-              containers are restarted.
-            </li>
-            <li><strong>Redis</strong>: Uses a persistent volume for Redis data (if required).</li>
-            <li><strong>Ports</strong>: Exposes only the necessary ports (e.g., 80 for the frontend and 8000 for the
-              backend).
-            </li>
-          </ul>
-        </div>
+// Komponenta za gumb v Sidebarju
+function SidebarButton({ item, activeTab, setActiveTab }: any) {

```

Comment:
```suggestion
const SidebarButton = ({ item, activeTab, setActiveTab }: any)  => {

```
pac pr teh je tut ta function... uredu ni nobene razlike sm fajn je da je povsot istu